dist/gcc-compatible/Hacl_Hash_SHA3.h

```diff
@@ -39,19 +39,23 @@ extern "C" {
 
 typedef Hacl_Streaming_MD_state_64 Hacl_Streaming_SHA3_state_256;
 
-Hacl_Streaming_MD_state_64 *Hacl_Streaming_SHA3_create_in_256(void);
+Hacl_Streaming_MD_state_64 *Hacl_Streaming_SHA3_malloc_256(void);
```

Comment:
What does the `256` in the name supposed to convey here?

Reply:
This comes from the current names.
Since they were discussions on SHA2 and SHA3 being updated soon we agreed to update their API afterwards.
The only modifications to this file are caused by the streaming functor update.